### ISSUE TITLE
Closing the program will prompt an error.

### ISSUE THREAD

**Issue by AyaseCore** [stoneharry/WoW-Spell-Editor#72]
I get an error when I close the program.
![image](https://user-images.githubusercontent.com/5546244/52965056-e177ec80-33de-11e9-8772-d1c97012247d.png)

I think this debug file is incomplete.
[debug_output.txt](https://github.com/stoneharry/WoW-Spell-Editor/files/2876616/debug_output.txt)

At least. It does not contain the following.
```
        // Ensure redirected console output is flushed
        ~MainWindow()
        {
            Console.WriteLine("######################################################");
            Console.WriteLine($"Stopped WoW Spell Editor - {DateTime.Now.ToString()}");
            Console.WriteLine("######################################################");
            Console.Out.Flush();
        }
```
So I think.The program is accessing a closed file(debug_output.txt).

Suggestion: 
1. I think the real-time output debugging information will be better. :)
2. Adds the current time to the debug information

**Comment by stoneharry** [comment]
Both good suggestions. I have not done either yet because I would have to update every Console.WriteLine in the code. :) I just have not got round to it.

It does not crash for me - but perhaps the order of events is not guaranteed and I am just not seeing the issue locally because of my environment. Redirecting the console output stream is a horrible solution.

**Comment by stoneharry** [comment]
I can reproduce this with the x86 version but not with the x64 version.

**Comment by stoneharry** [comment]
Fixed in v1.9.4 - closing.
